x/sigs/codec.proto

```diff
@@ -25,3 +25,12 @@ message StdSignature {
   // Removed Address, Pubkey is more powerful
   crypto.Signature signature = 4;
 }
+
+// BumpSequenceMsg increments a sequence counter by given amount for a user.
+message BumpSequenceMsg {
+  // Public key of the user.
+  crypto.PublicKey pubkey = 1;
+  // Increment represents the value by which a sequence value will be
+  // increased.
+  int64 increment = 2;
```

Comment:
req: let's make this uint32 . It must not be negative and within the upper limit
pp: please add documentation of the allowed range.